### DISCUSSION TITLE
docker run --rm -v $(pwd):/src -w /src kjconroy/sqlc generate not working on windows 10 OS

### DISCUSSION THREAD

**Discussion by rassidm** [sqlc-dev/sqlc#880] [Q&A] [answered]
docker run --rm -v $(pwd):/src -w /src kjconroy/sqlc generate not working on windows 10 OS

I tried :

docker run --rm -v %cd% -w /src kjconroy/sqlc generate
not working 
my sqlc.yaml is in current directory same path returned by %cd%

**Comment by kyleconroy** [comment]
Can you provide a bit more information? When you say "not working", what do you mean? Is there any output? Is there an error message?

**Comment by rassidm** [comment]
Please add a way that we can install SQLC on Windows 10 using **chocolatey** or **Scoop**
Thank you very much for your Great work :)

**Comment by kyleconroy** [comment, marked as answer]
I added Windows support (https://github.com/kyleconroy/sqlc/pull/886) here, which means you can now [download tip releases for Windows](https://docs.sqlc.dev/en/latest/overview/install.html#tip-releases). A huge caveat: these builds do not support the PostgreSQL engine.

**Comment by xiaomingdebaba** [comment]
i tried : docker run --rm -v D:\simplebank:/src -w /src kjconroy/sqlc generate

return this :# package db
error parsing queries: path /src/db/query does not exist
Makefile:17: recipe for target 'sqlc' failed

how to solve

**Reply by abhishek-hotstar** [reply]
docker run --rm \
    	-v ${PWD}:/src \
    	-w /src \
    	kjconroy/sqlc \
    	generate

**Reply by go-aegian** [reply]
if ${PWD} does not work these below do

` -v "$(pwd)":/src `

or with subfolders

`
 -v "$(pwd)"/path1/path2:/src `
